src/components/Calendar/src/Calendar.vue

```diff
@@ -402,7 +356,6 @@ export default {
 	overflow: hidden;
 	color: var(--color-black);
 	text-align: center;
-	cursor: pointer;
```

Comment:
Since the disabled state was moved to the button-element inside of this table-cell, the `cursor: pointer` should also be moved down, otherwise disabled button _table-cells_ will still have pointer cursors.